packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-xray-sdk-core",
-  "version": "2.2.0-experimental.1",
+  "version": "2.3.1-experimental.1",
```

Comment:
Does the package version independent from the version listed in CHANGELOG? Just curious about it since I saw the version listed in changelog is different from this one.

Reply:
In this case you can see this as version 2.3.1, just with the experimental tag. I haven't been marking the changelog versions as experimental on this branch for the past few releases, but I can if you think that'll be more clear.

Reply:
That's fine, just want to make sure we don't mess up with the version since I am not familiar with how experimental works. Thank you for your explanation. :)